packages/babel-node/src/_babel-node.ts

```diff
@@ -16,7 +16,7 @@ import type { PluginAPI, PluginObject } from "@babel/core";
 
 const require = createRequire(import.meta.url);
 
-const program = new commander.Command("babel-node");
+const program = commander.default.program as commander.Command;
```

Comment:
Here we get the program from the commander default export so that it works for Babel 8 with commander 12.0 as well.